sandbox/apps/sandbox/fields.js

```diff
@@ -95,7 +90,6 @@ module.exports = {
     mixin: 'textarea',
     // we want to ignore default formatters as we want
     // to preserve white space
-    isPageHeading: 'true',
     'ignore-defaults': true,
     // apply the other default formatters
     formatter: ['trim', 'hyphens'],
```

Comment:
have you added a maxword to the sandbox app?

Reply:
I have added in a max word example to the sandbox app now